template/lambda-sqs-worker-cdk/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # syntax=docker/dockerfile:1.10
 
-FROM --platform=<%- platformName %> node:20-alpine AS dev-deps
+FROM --platform=<%- platformName %> public.ecr.aws/docker/library/node:20-alpine AS dev-deps
```

Comment:
scope creep but should we be ditching --platform stuff given the docker warnings? @72636c

Reply:
I'm comfortable with that 🚫🪟

Reply:
Donezo